app/schemas/data_source.py

```diff
@@ -28,5 +28,16 @@ class Config:
         orm_mode = True
 
 
+class DataSourceInDBIDOnly(DataSourceBase):
```

Comment:
I think we don't need `DataSourceInDB` and `DataSource` anymore. Can you remove them and then you can reuse their names for the new models.

Reply:
Yup, removed them in the new commit